runway/hooks/staticsite/auth_at_edge/templates/parse_auth/__init__.py

```diff
@@ -0,0 +1,138 @@
+# pylint: skip-file
+"""Parse the given Cognito/Local authorization information.
+
+Ensure the parsed Cognito and Local authorization information from the retrieved
+query string parameter and cookie headers matches our expectations. If it doesn't
+then inform the user of a bad request, otherwise retrieve the Cognito tokens to
+add to the cookie headers.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+
+import json
+import logging
+import traceback
+import base64
+from urllib.parse import parse_qs
+
+from shared import (create_error_html, extract_and_parse_cookies, get_config,  # noqa pylint: disable=import-error
+                    get_cookie_headers, http_post_with_retry)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def handler(event, _context):
+    """Handle the authorization parsing.
+
+    Keyword Args:
+        event (Any): The Lambda Event
+    """
+    request = event['Records'][0]['cf']['request']
+    config = get_config()
```

Comment:
Same comment as above -- get_config() instantiation should be able to occur outside the handler